test/classes/UserTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+require __DIR__ . '/../../classes/User.php';
```

Comment:
This should work without explicitly requiring the class(as that's the autoloaders job)